app/adapter/web/src/component/pages/shared/SignIn.tsx

```diff
@@ -16,16 +17,9 @@ export class SignIn extends Component<Props> {
           Please sign in with Github so that all the short links created can be
           linked to your account.
         </div>
-        <a href={this.props.githubSignInLink}>
-          <div className={'button github'}>
-            <img
-              alt={'Sign in with github account'}
-              className={'icon'}
-              src={githubLogo}
-            />{' '}
-            Sign In with Github
-          </div>
-        </a>
+        <GithubSignInButton signInLink={this.props.githubSignInLink} />
+        or
```

Comment:
Maybe I'm too concerned about such things, but I think word "or" is a bit superfluous here

Reply:
Agreed.